js/pulse.js

```diff
@@ -806,7 +806,8 @@
          */
         this.sharedElementGive = function() {
             setTimeout(function(){
-                _adManager.ui.ooyalaVideoElement[0].play();
+                _adManager.sharedVideoElement.style.display = "block";
+                _adManager.sharedVideoElement.play();
             }, 100);
             _adManager.sharedVideoElement.style.visibility ="hidden";
```

Comment:
@pilievOoyala I think the plugin shouldn't need to set visibility since vtc is doing that - are you able to verify this?